README.md

```diff
@@ -1,2 +1,92 @@
-# ibm-ingress-nginx-operator
-operator for ingress nginx
+# IBM Ingress nginx Operator
```

Comment:
For the readme (not CSV) the title should be 
# ibm-ingress-nginx-operator

Reply:
Updated.